src/common/m_derived_types.fpp

```diff
@@ -168,6 +168,7 @@ module m_derived_types
         !! patch geometries. It is specified through its x-, y-, and z-components
         !! respectively.
 
+        type(ic_model_parameters) :: model !<
```

Comment:
are you using this anywhere? i'm not sure why it was added but I could be ignorant

Reply:
Removed in the upcoming comment.